src/per_task.c

```diff
@@ -263,14 +247,19 @@ static ui_handlrp_t handle_term_inp(void)
     {
       if (key == (char)_GET_KEY_CODE( n ))
       {
-// any terminal output specific to the key or handelr need to be done here and
+// any terminal output specific to the key or handler needs to be done here and
 // not in the handler itself because the handler is to be called from w/i the CS
         fp =_GET_UI_HDLRP( n );
         break;
       }
     }
 // anykey ...
-    Log_Level = 255;// default anykey enable continous/verbous log
+    Log_Level = 255;// by default, any-key enables continous/verbose log
+    if (Radio_detect_timer < KEYBOARD_DETECT_WINDOW)
+    {
+      Enable_radio_input = FALSE;
```

Comment:
For the 3 variables written here and are accessed via interrupt and non-interrupt functions, consider declaring them as volatile to turn off optimization on these variables.